### ISSUE TITLE
Item list ordering is illogical (to me)

### ISSUE THREAD

**Issue by miigotu** [plexinc/plex-media-player#501]
Whether it is a show/movie list sorted alphabetically, seasons sorted by season number, or episodes sorted by episode number, the list is displayed top row, bottom row, next column top row, bottom row, and so on. In other words, "odd" on top, "even" on bottom.

This results in a sorting grid like this:
|1|3|5|7|
|2|4|6|8|

One would normally expect to read from left to right (or in some languages right to left), but never would naturally follow this pattern.

I think it should go:

|1|2|3|4|
|5|6|7|8|

Thoughts?

**Comment by Skaronator** [comment]
You're completely correct in theory, but in practice, the current path is the better way. For example if you split a letter over 2 rows:
```
|A|A|A|A|B|B|
|B|C|D|E|E|E|
```
or when you are searching something it would be easy if it's in the 1st row but if it's on the 2nd row it would be not that good.


A better way at all would be if they make the scrolling vertical not horizontal and then:

|1|2|3|4|5|6|
|7|8|9|0|1|2|

**Comment by miigotu** [comment]
When you are operating with a remote, which is most of the time with plex, having to go down up right down up right down up right to follow the order doesnt make sense at all, regardless if the split is in the middle of a letter or not when going in a natural order.  If anything when you press right on the last item in a row it should move you to the start of the next row to continue the order.  1 button press to move ahead one seas, never 2.

zig-zagging the order really  no sense at all. It makes it almost impossible to know where you are going without having to read the text, especially in picking a show season when you would expect to go one step to the next season, but if you are on say season 3 moving one step right will land you on season 5.... you have to go up and then right (2 moves to change ahead one season)......

**Comment by Skaronator** [comment]
Huch? Why do you need to "hover" over each item? For me, the cover of the item's "says enough". You could also try out the list view.

But Plex should also add a vertical view for the poster so you can switch between `Poster (horizontal)` and `Poster (vertical)`

**Comment by miigotu** [comment]
When the next season is always to the right you dont even have to look, from the end of one season when the last episode is over its just `back`, `right`, `select`.

With how it is now you don't know if you need to go `back`, `down`, `select` or `back`, `up`, `right`, `select` .... So you have to try and hunt down the text on the poster (which isnt always so easy) or wait for the season info to show up in the bottom left of the screen, AFTER selecting it.

It makes you take longer switching seasons, and annoys you in the process.

**Comment by Skaronator** [comment]
Oh I totally forgot about the season. Right this one is really bad indeed.

**Comment by Skaronator** [comment]
Do you know what happens when you have over 10 seasons? Does it starts to scroll horizontal?

**Comment by miigotu** [comment]
Dont know, I only keep a few seasons at a time

**Comment by Skaronator** [comment]
Found a nice reference to the current layout on the Material UI docs (https://material.io/guidelines/components/grid-lists.html#grid-lists-behavior)

> Horizontally scrolling grid lists are discouraged because the scrolling interferes with typical reading patterns, affecting comprehension. One notable exception is a horizontally-scrolling, single-line grid list of images, such as a gallery, which is compatible with typical reading patterns.

**Comment by miigotu** [comment]
That reference does not describe the current layout in plex whatsoever. It describes either horizontal OR vertical, not a flip-flop of vert-horiz-vert-horiz like there is in plex atm.

**Comment by miigotu** [comment]
"Tile filtering and sorting

Content in a grid list can be programmatically filtered or sorted by date, file size, alphabetical order, or other parameters.

The first item in the grid list is positioned at the top left of the grid list, and the order proceeds left to right and top to bottom."
My point. Left to right top to bottom means you go from top left to top right, then you start on the next line on the far left and __read__ to the right... just like a book.

**Comment by LongChair** [comment]
@miigotu : as most of the interface changed with v2 can we close this ?

**Comment by miigotu** [comment]
The newest PMP release I can view on rpi3 with plexpass update channel is 1.3.11.758-4be89b5c, which still has the issue. If there is a way for me to see the updated interface on rpi3, please let me know, otherwise I think this should be left open until the fix can be verified.

**Comment by mseeley** [comment]
Hi @miigotu thanks for your feedback.

We're focusing our efforts on shipping our 2.x interface. The order of items in the 1.x interface won't be receiving an update.
